src/renderer/containers/Login/LoginContainer.tsx

```diff
@@ -1,28 +1,53 @@
 import * as React from "react";
-import {ReactNode} from "react";
+import {ReactElement, Component} from "react";
 import {Background} from "../../components/Background/Background";
 import {Modal} from "../../components/Modal/Modal";
 import {InputForm} from "../../components/Input/InputForm";
 import {ButtonPrimary, ButtonSecondary} from "../../components/Button/ButtonStandard";
 import {Link} from "react-router-dom";
 import {Routes, OnBoardingRoutes} from "../../constants/routes";
+import {RouteComponentProps} from "react-router";
+import {Notification} from "../../components/Notification/Notification";
+import {Level, Horizontal, Vertical} from "../../components/Notification/NotificationEnums";
+import database from "../../services/db/api/database";
+import {bindActionCreators, Dispatch} from "redux";
+import {connect} from "react-redux";
+import {storeAuthAction,} from "../../actions";
+import {IRootState} from "../../reducers";
 
 interface IState {
     input: string;
+    notification: boolean;
+}
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface IOwnProps extends Pick<RouteComponentProps, "history"> {
+}
```

Comment:
You could avoid eslit error with
`type IOwnProps = Pick<RouteComponentProps, "history">`